apps/services/auth/admin-api/src/app/v2/clients/clients.module.ts

```diff
@@ -1,15 +1,20 @@
 import { Module } from '@nestjs/common'
+import { SequelizeModule } from '@nestjs/sequelize'
 
 import {
+  Client,
   ClientsModule as ClientsLibModule,
   ResourcesModule as ResourcesLibModule,
 } from '@island.is/auth-api-lib'
 
 import { MeClientsController } from './me-clients.controller'
 
 @Module({
-  imports: [ClientsLibModule, ResourcesLibModule],
+  imports: [
+    SequelizeModule.forFeature([Client]),
```

Comment:
Isn´t this leftover clean up after the secrets moved to seperate module?

Reply:
Indeed, removed.